atc/db/build.go

```diff
@@ -1246,6 +1254,17 @@ func (b *build) AdoptRerunInputsAndPipes() ([]BuildInput, bool, error) {
 		return nil, false, err
 	}
 
+	_, err = psql.Update("builds").
```

Comment:
This seems out-of-place in this PR; is it fixing something? Should it be a separate commit + test?